Bank-admin-API-design.md

```diff
@@ -0,0 +1,55 @@
+# Bank admin API
+
+## /bank_admin/transactions/ - GET
+
+Returns a list of all recent transactions
+
+### Filters
+
+**status**
+
+Filters by the status of the transactions, accepts multiple values which will be 
+OR'd.
+
+Possible values:
+- *available*: returns list of transactions that have not been locked by anyone
+- *locked*: returns list of locked transactions
+- *credited*: returns list of transactions credited
+- *refunded* (*not implemented yet*): returns list of transactions refunded
+- *refund_pending* (*not implemented yet*): returns list of transactions in pending refund
+
+For ADI file generation, this will likely be:
+
+?status=credited&status=refunded
```

Comment:
Wait, are we saying `&` is treated at `OR`? That seems counter-intuitive? I'd suggest we treat `&` as `AND`, and use comma delimiting for `OR` (i.e. `?status=credited,refunded`).